apps/api/src/queries/buildViewModelForResource/viewModels/vocabulary-list.view-model.ts

```diff
@@ -0,0 +1,200 @@
+import {
+    AggregateType,
+    ContributorWithId,
+    FormFieldType,
+    IDynamicForm,
+    IFormField,
+    IMultilingualText,
+} from '@coscrad/api-interfaces';
+import { isNonEmptyObject } from '@coscrad/validation-constraints';
+import { ApiProperty } from '@nestjs/swagger';
+import { DetailScopedCommandWriteContext } from '../../../app/controllers/command/services/command-info-service';
+import { ICoscradEvent } from '../../../domain/common';
+import { buildMultilingualTextWithSingleItem } from '../../../domain/common/build-multilingual-text-with-single-item';
+import { MultilingualText } from '../../../domain/common/entities/multilingual-text';
+import BaseDomainModel from '../../../domain/models/base-domain-model.entity';
+import { AccessControlList } from '../../../domain/models/shared/access-control/access-control-list.entity';
+import {
+    FilterPropertyType,
+    VocabularyListCreated,
+    VocabularyListFilterPropertyRegistered,
+} from '../../../domain/models/vocabulary-list/commands';
+import { AggregateId } from '../../../domain/types/AggregateId';
+import { HasAggregateId } from '../../../domain/types/HasAggregateId';
+import { DTO } from '../../../types/DTO';
+import { TermViewModel } from './term.view-model';
+
+export class VocabularyListEntryViewModel extends BaseDomainModel {
+    // note this doesn't quite line up with the `IVocabularyListViewModel` interface
```

Comment:
It will be interesting to see how to deal with this in the response interceptor.